Whiskey/en-US/about_Whiskey_NUnit3_Task.help.txt

```diff
@@ -29,6 +29,8 @@ PROPERTIES
     * `OpenCoverArgument`: a list of additional arguments to pass to the OpenCover console.
     * `ReportGeneratorVersion`: the version of ReportGenerator to use. Defaults to the latest version available.
     * `ReportGeneratorArgument`: a list of additional arguments to pass to the ReportGenerator console.
+    * `SupportNUnit2`: boolean indicating if NUnit2 test support should be enabled. Defaults to false and is only intended for old non-migrated NUnit2 test projects.

```

Comment:
Add a note that using this option will download additional packages that include NUnit2 support.